trailblazer/store/crud/update.py

```diff
@@ -26,10 +26,12 @@
 class UpdateHandler(BaseHandler):
     """Class for updating items in the database."""
 
-    def update_analysis_jobs(self, analysis: Analysis, jobs: list[dict]) -> None:
+    def update_analysis_jobs(self, jobs: list[dict]) -> None:
         """Update jobs in the analysis."""
-        analysis.jobs = [Job(**job) for job in jobs]
         session: Session = get_session()
+        for job in jobs:
+            updated_job = Job(**job)
+            session.add(updated_job)
```

Comment:
Will this also add them to analysis.jobs? Seems like we are changing the logic here by not setting that?

Reply:
And if they are added via the analysis.id we will just add them on top of the old ones, rather than switching?

Reply:
Yes, the job dictionary contains the analysis id. And no, they are deleted before this function is called. Would probably make sense to delete them in here instead, but I'm minimizing the changes for now